### DISCUSSION TITLE
How do I cache (NuGet) a previously installed port?

### DISCUSSION THREAD

**Discussion by funnan-dev** [microsoft/vcpkg#27966] [Q&A]
I have 300+ ports I have built and cached using `--binarysource=nuget,<MyRepo>,readwrite`. Some of the built packages failed to push to the server due to the timeout being hit. I would like to make another attempt at pushing the failed packages to the server but I can't find the previous nuget packages generated by vcpkg (I suspect they were deleted by vcpkg).

How do I use vcpkg to cache these previously installed ports that failed to cache their first time around?

**Comment by JackBoosY** [comment]
You can get the default cache path in this doc:
https://github.com/microsoft/vcpkg/blob/master/docs/users/binarycaching.md#configuration
